src/Evolution/DiscontinuousGalerkin/Limiters/Minmod.hpp

```diff
@@ -220,7 +220,7 @@ class Minmod<VolumeDim, tmpl::list<Tags...>> {
   /// \param element_size The size of the element in inertial coordinates, along
   ///        each dimension of logical coordinates.
   /// \param orientation_map The orientation of the neighbor
-  void package_data(const gsl::not_null<PackagedData*>& packaged_data,
+  void package_data(gsl::not_null<PackagedData*> packaged_data,
```

Comment:
why is this being changed?

Reply:
Because there is little benefit to taking the pointer by reference. Nils told me to make this change in other code, and I am doing it here too for consistency.